test/test_unary_ufuncs.py

```diff
@@ -19,7 +19,7 @@
     onlyCUDA, dtypesIfCUDA, precisionOverride, skipCUDAIfRocm, dtypesIfCPU,
     OpDTypes)
 from torch.testing import (
-    floating_types_and, all_types_and_complex_and, floating_types)
+    floating_types_and, all_types_and_complex_and, floating_types, get_all_complex_dtypes, get_all_fp_dtypes)
```

Comment:
Let's continue to use the type functions corresponding to the dispatch macros. In fact, we should probably add a note here:

https://github.com/pytorch/pytorch/blob/febc76a5c69302c7d4b481632bcbd25e5d9b76ca/torch/testing/__init__.py#L325

that the dtype functions corresponding to the dispatch macros are preferred to the "get_all..." functions.

Reply:
Why deprecating these `get_all` things? They are the cleanest thing because their `all` really means all. In comparison, floating_types is just fp32 and fp64 but not bf16 or half, which doesn't look to make sense to me at all. The dispatch macros already suck, why making the testing suite worse by copying that behavior?